src/sentry/utils/sentry_apps/webhooks.py

```diff
@@ -126,69 +126,83 @@ def send_and_save_webhook_request(
     :param url: The URL to hit for this webhook if it is different from `sentry_app.webhook_url`.
     :return: Webhook response
     """
-    buffer = SentryAppWebhookRequestsBuffer(sentry_app)
+    from sentry.sentry_apps.metrics import SentryAppInteractionEvent, SentryAppInteractionType
```

Comment:
is this here because you get a circular import otherwise?

Reply:
nodders